include/class.config.php

```diff
@@ -767,6 +767,10 @@ function getDefaultTicketSequence() {
         return $sequence;
     }
 
+    function getTopLevelTicketCounts() {
```

Comment:
In the spirit of making it consistent - I think this function should be `showTopLevelTicketsCounts()` and `return ($this->get('ticket_counts'));`